src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -530,9 +526,7 @@ describe('NewWorkspaceModal', () => {
 
       // Assert
       screen.getByText(policyTitle);
-      screen.getByText(policyLabel);
-      // Informational links that should render
-      screen.getByRole('link', { name: 'Learn more about policies' });
+      // Informational link about cost.
```

Comment:
The informational link about policies is tested in `WorkspacePolicies.test.tsx` now.